wildfly/setCheckedActionFactoryExample/pom.xml

```diff
@@ -23,17 +23,16 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
 	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>narayana-quickstarts-all</artifactId>
+        <groupId>org.jboss.narayana.quickstart</groupId>
+        <version>5.13.1.Final-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
```

Comment:
Same here

Reply:
Thanks @jmfinelli ! Actually in this case it is not the default therefore I would leave it like that.

Reply:
Sorry! My copy and paste error ;-)